src/app/app.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { TestBed } from '@angular/core/testing';
```

Comment:
Please add this boilerplate in this file and all source codes

```
// Copyright 2022 The Triple Banana Authors. All rights reserved.
// This Source Code Form is subject to the terms of the Mozilla Public
// License, v. 2.0. If a copy of the MPL was not distributed with this
// file, You can obtain one at https://mozilla.org/MPL/2.0/.
```

Reply:
will add it to app.component.spec.ts and app.component.ts